libtelemetry/gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=4.0.0-SNAPSHOT
+VERSION_NAME=3.1.0-SNAPSHOT
```

Comment:
This could be perfectly `4.0.0-SNAPSHOT`

Reply:
OK I'm unclear on this. What api breaking change does this do? It is a feature tweak at best. I'm assuming we are going to release many releases before 4.0.0. So I need explanation on your thought process here, because to me, that's jumping the gun.

Reply:
Changing namespaces is a major version because developers need to change their code (pointing to the new package).

Besides that, we normally bump the snapshot version name after a release has happened. It's a way of telling devs what's cooking for the next major version 👀 https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/gradle.properties#L2

We could definitely keep releasing 3.x versions in the meantime (note that this is only for `SNAPSHOT`s).

Does that make sense?

Reply:
👍